mne/io/kit/coreg.py

```diff
@@ -70,8 +70,9 @@ def read_mrk(fname):
     elif fname.suffix == ".txt":
         mrk_points = _read_dig_kit(fname, unit="m")
     elif fname.suffix == ".pickled":
+        # TODO should we still be supporting pickled here?
         with open(fname, "rb") as fid:
-            food = pickle.load(fid)
+            food = pickle.load(fid)  # nosec B301
```

Comment:
I would say no. Let's deprecate and see if people complain and we'll make sure some better stable public interface works (like writing to text or FIF).